internal/pkg/plugin/zentao/zentao.go

```diff
@@ -0,0 +1 @@
+package zentao
```

Comment:
No newline at end of file. Please do a 'go fmt'.

Reply:
I'll fix it.